library/src/main/AndroidManifest.xml

```diff
@@ -8,9 +8,11 @@
             android:launchMode="singleTask"
             android:taskAffinity="com.chuckerteam.chucker.task"
             android:theme="@style/Chucker.Theme" />
+
         <activity android:name="com.chuckerteam.chucker.internal.ui.transaction.TransactionActivity"
             android:theme="@style/Chucker.Theme"
             android:parentActivityName="com.chuckerteam.chucker.internal.ui.MainActivity"/>
+
```

Comment:
nit: whitespace change here that can be reverted

Reply:
I`ve added it for readability.